src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -1207,7 +1209,56 @@ CHIP_ERROR GetNumberOfCertsFromPKCS7(const char * pkcs7, uint32_t * n_certs)
 CHIP_ERROR ValidateCertificateChain(const uint8_t * rootCertificate, size_t rootCertificateLen, const uint8_t * caCertificate,
```

Comment:
caCertificate should be renamed icaCertificate. Suggest opening an issue in the repos and assigning it to me. Not needed in this PR.

Reply:
done [#9487].   Couldn't "assign" it.